airbyte-local.sh

```diff
@@ -849,8 +848,22 @@ function cleanup() {
             docker container kill $(cat "$tempPrefix-dst_cid") 2>/dev/null || true
             rm "$tempPrefix-dst_cid"
         fi
+        src_exit_code=$(docker inspect $src_container_name --format="{{.State.ExitCode}}")
+        dst_exit_code=$(docker inspect $dst_container_name --format="{{.State.ExitCode}}")
+        if ((keep_containers)); then
+            log "Docker containers $src_container_name and $dst_container_name have been saved"
+        else
+            docker container rm -f $src_container_name $dst_container_name > /dev/null 2>&1 || true
+        fi
     fi
     rm -rf "$tempdir"
+    if [ -n "$src_exit_code" ] && [ "$src_exit_code" -ne 0 ]; then
```

Comment:
I am thinking if we should always check both exit codes, print messages in each case always, and then exit if either of the codes are non zero?

Reply:
Done